app/src/main/java/de/tum/in/tumcampusapp/services/JobIdConstants.java

```diff
@@ -0,0 +1,16 @@
+package de.tum.in.tumcampusapp.services;
+
+/**
+ * Created by ansel on 11/11/2017.
+ */
```

Comment:
Please don't commit the autogenerated comments. A small description of the class would be awesome 👍